components/raftstore/src/store/fsm/apply.rs

```diff
@@ -941,7 +959,7 @@ impl ApplyDelegate {
             );
         }
 
-        if cmd.has_admin_request() {
+        if should_write_to_engine_after(&cmd) {
```

Comment:
Why not set it in `handle_ingest_sst`?

Reply:
Follow the same style as `should_write_to_engine_before`.